scripts/update/bash_completion.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+#
+# Source the bash_completion installer
+. /etc/swizzin/sources/functions/bash_completion
```

Comment:
Nothing being sourced here
```suggestion
bash /etc/swizzin/sources/functions/bash_completion
```